src/component/mxgraph/style/utils.ts

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 
-import type { GlobalTaskKind, ShapeBpmnSubProcessKind } from '../../../model/bpmn/internal';
+import type { GlobalTaskKind, MessageVisibleKind, ShapeBpmnSubProcessKind } from '../../../model/bpmn/internal';
```

Comment:
ℹ️ not related to the fix, but I didn't want to create a dedicated PR for this 😸